goserver/session/session.go

```diff
@@ -0,0 +1,96 @@
+package session
+
+import (
+	"crypto/hmac"
+	"crypto/sha1"
+	"encoding/base64"
+	"flag"
+	"fmt"
+	"net/http"
+	"net/url"
+	"strings"
+	"time"
+)
+
+var sessionCookieSecret = flag.String("session_cookie_secret", "", "secret key used to encrypt the session cookie")
```

Comment:
Flags shouldn't be used to configure packages. Flags should be defined in main and passed to objects.

Reply:
See the new way of getting SessionCookieSecret in session. Is this a common way?